### ISSUE TITLE
mat-select ngModelChange/selectionChange triggers twice on selecting option with undefined value

### ISSUE THREAD

**Issue by JGH153** [angular/material2#12684]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Triggering/emitting only once. 

#### What is the current behavior?
If you select an option without a value propery (thus making it undefined) ngModelChange/selectionChange triggers twice. 

#### What are the steps to reproduce?
https://stackblitz.com/angular/bnolvjbmlro

#### What is the use-case or motivation for changing an existing behavior?
Inconsistent behavior

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular CLI: 6.1.2
Node: 8.9.1
OS: win32 x64
Angular: 6.1.2
Material: 6.4.5

#### Is there anything else we should know?
No

**Comment by crisbeto** [comment]
Closing as a duplicate of https://github.com/angular/material2/issues/10675.
